app/src/org/commcare/activities/CommCareActivity.java

```diff
@@ -635,7 +635,7 @@ protected void tryToAddActionSearchBar(Activity act, Menu menu,
                                            ActionBarInstantiator instantiator) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
             MenuInflater inflater = act.getMenuInflater();
-            inflater.inflate(org.commcare.dalvik.R.menu.activity_report_problem, menu);
+            inflater.inflate(org.commcare.dalvik.R.menu.action_bar_search_view, menu);

```

Comment:
I'm confused how activity_report_problem became action_bar_search_view without any actual changes to the file